app/controllers/users_controller.rb

```diff
@@ -0,0 +1,10 @@
+class UsersController < ApplicationController
+	def update
+		@user = User.find(params[:id])
+		@user.email_notification_enabled = !@user.email_notification_enabled 
+		@user.save
+		respond_to do |format|
+			format.js {flash.now[:notice] = "Email Notification Changed Successfully"}
+		end
+	end
```

Comment:
would prefer smaller spacing (2 spaces = 1 tab)

Reply:
spacing is fine locally. Must just be an issue with on GH or something